modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -3,6 +3,9 @@
 
 export const BASE_PATH = "{{{basePath}}}".replace(/\/+$/, "");
 
+const isBlob = (value: any) => Blob !== undefined && value instanceof Blob;
```

Comment:
```suggestion
const isBlob = (value: any) => typeof Blob !== 'undefined' && value instanceof Blob;
```
since Blob is not necessarily a declared variable

Reply:
see e.g. https://stackoverflow.com/a/4725697